packages/core-design-system/scripts/build.js

```diff
@@ -0,0 +1,304 @@
+    file => !file.startsWith(THEME_MODE_FILENAME_PREFIX.DARK) && !file.startsWith(THEME_MODE_FILENAME_PREFIX.LIGHT)
+  )
+  const ossDarkFiles = cssFiles.filter(
+    file => file.includes(OSS_STYLES_SOURCE_NAME) && file.startsWith(THEME_MODE_FILENAME_PREFIX.DARK)
+  )
+  const ossLightFiles = cssFiles.filter(
+    file => file.includes(OSS_STYLES_SOURCE_NAME) && file.startsWith(THEME_MODE_FILENAME_PREFIX.LIGHT)
+  )
+
+  console.log('\n=== Theme File Summary (OSS) ===')
+  console.table([
+    { Type: 'Dark Theme Files', Count: ossDarkFiles.length },
+    { Type: 'Light Theme Files', Count: ossLightFiles.length }
+  ])
+  console.log('\n')
+
+  /**
+   * Core imports
+   * */
+  const coreStyles = `${getExportFileHeader()}
+   
+/* Core tokens */
+${coreFiles.map(file => `@import './${file}';`).join('\n')}`
+
+  /**
+   * OSS themes imports
+   * */
+  const ossContent = `${getExportFileHeader()}
+
+/* Theme files - Dark */
+${ossDarkFiles.map(file => `@import './${file}';`).join('\n')}
+
+/* Theme files - Light */
+${ossLightFiles.map(file => `@import './${file}';`).join('\n')}`
+
+  /**
+   * Enterprise themes imports
+   * */
+  const enterpriseContent = `${getExportFileHeader()}
+
+/* Theme files - Dark */
+@import './dark-harness.css';
+
+/* Theme files - Light */
+@import './light-harness.css';`
+
+  // Write file
+  await Promise.all([
+    fs.writeFile(`${DESIGN_SYSTEM_ROOT}/oss.css`, ossContent),
+    fs.writeFile(`${DESIGN_SYSTEM_ROOT}/enterprise.css`, enterpriseContent),
+    fs.writeFile(`${DESIGN_SYSTEM_ROOT}/core-imports.css`, coreStyles)
+  ])
+
+  console.log('\n\x1b[1m\x1b[32m%s\x1b[0m', `✔︎ Created import files in ${DESIGN_SYSTEM_ROOT}`)
+}
+
+async function createEsmIndexFile() {
+  console.log(`\n\x1b[34mCreating ${DESIGN_SYSTEM_ROOT_ESM}/index.ts import file...\x1b[0m`)
+
+  // Get list of all JS files
+  const styleValueFiles = (await fs.readdir(DESIGN_SYSTEM_ROOT_ESM))
+    .filter(file => file.endsWith('.ts') && file !== 'index.ts')
+    .sort()
+
+  // Organize files by type
+  const coreFiles = styleValueFiles.filter(
+    file => !file.startsWith(THEME_MODE_FILENAME_PREFIX.DARK) && !file.startsWith(THEME_MODE_FILENAME_PREFIX.LIGHT)
+  )
+  const darkFiles = styleValueFiles.filter(file => file.startsWith(THEME_MODE_FILENAME_PREFIX.DARK))
+  const lightFiles = styleValueFiles.filter(file => file.startsWith(THEME_MODE_FILENAME_PREFIX.LIGHT))
+
+  console.log('\n=== Theme File Summary (ts) ===')
+
+  console.table([
+    { Type: 'Dark Theme Files', Count: darkFiles.length },
+    { Type: 'Light Theme Files', Count: lightFiles.length }
+  ])
+  console.log('\n')
+
+  // Generate content
+  const content = `/**
+ * Harness Design System
+ * DO NOT UPDATE IT MANUALLY
+ * Generated on ${new Date().toUTCString()}
+ */
+
+  /* Theme files - Combined */
+export const designSystemThemeMap = {
+${[...darkFiles, ...lightFiles]
+  .map(file => {
+    const name = file.replace('.ts', '')
+    return `'${name}': '${name}',`
```

Comment:
Just curious why we'd use an object with the same values repeated for the name and value. Why not just have an array of `name`? It doesn't appear to be used anywhere, so not sure of its intended use 🤔